src/ui/components/map/providers/mapprovider.js

```diff
@@ -90,6 +90,19 @@ export default class MapProvider {
      * @type {boolean}
      */
     this._collapsePins = config.collapsePins || false;
+
+    /**
+     * Locale of the map. MapComponent supplies the locale specifed by
+     * ANSWERS.init() by default
+     * @type {string}
+     */
+    const locale = config.locale;
+
+    /**
+     * Language of the map. Defaults to 'en'
+     * @type {string}
+     */
+    this._language = locale.substring(0, 2) || 'en';
```

Comment:
Is the language always going to be the substring from 0 to 2? I'm used to seeing it parsed by getting the substring until - or _ but I have no idea what the rules are for language codes 🙈

Reply:
Yep, there are few specifications for locale strings, but the specs that I've seen always require a language as the first two characters. Both Google and Mapbox support these codes. This will miss a couple locales that Google supports that include region (e.g. en-GB or fr-CA). I can check with product, and if they want that functionality I can tweak this to support that

Reply:
Updating to support these edge cases